tests/suites/test_suite_psa_its.function

```diff
@@ -143,11 +152,11 @@ void set_multiple( int first_id, int count )
     {
         mbedtls_snprintf( stored, sizeof( stored ),
                           "Content of file 0x%08lx", (unsigned long) uid );
-        PSA_ASSERT( psa_its_get( uid, 0, sizeof( stored ), retrieved ) );
+        PSA_ASSERT( psa_its_get( uid, 0, sizeof( stored ), retrieved, &ret_len ) );
         ASSERT_COMPARE( retrieved, sizeof( stored ),
                         stored, sizeof( stored ) );
         PSA_ASSERT( psa_its_remove( uid ) );
-        TEST_ASSERT( psa_its_get( uid, 0, 0, NULL ) ==
+        TEST_ASSERT( psa_its_get( uid, 0, 0, NULL, NULL ) ==
```

Comment:
Good to see that we are also testing the `NULL` `data_length` parameter case.

Reply:
Ok